schema/pet-insurance.json

```diff
@@ -288,25 +284,20 @@
             "minItems": 1,
             "maxItems": 3,
             "items": {
-                "$ref": "#/domains/PetInsurance/types/VoluntaryExcessItem"
+                "$ref": "#/domains/PetInsurance/types/VoluntaryExcess"

```

Comment:
Why is `maxItems` 3? Can't there be 4 voluntary excess options on a different website?